controllers/numaresourcesoperator_controller_test.go

```diff
@@ -77,6 +81,7 @@ func NewFakeNUMAResourcesOperatorReconciler(plat platform.Platform, initObjects
 		Helper:       helper,
 		Namespace:    testNamespace,
 		ImageSpec:    testImageSpec,
+		Recorder:     recorder,
```

Comment:
this is correct, but is something missing here?

Reply:
I followed the convention from here:https://github.com/openshift-kni/numaresources-operator/blob/main/controllers/kubeletconfig_controller_test.go#L50

Reply:
ok, in both cases we are not actually testing the events. Good enough for now.

Reply:
I was trying to test the actual events, but it seems like this fake recorder, doesn't actually create the event (```client.List(ctx, eventList...)``` returns empty), but at least it allows you to run the tests without crashing when calling the recorder.

Reply:
Here you can see how you can check event
https://github.com/openshift-kni/performance-addon-operators/blob/e5294c90a8e103d19318d3cf035c132b3052d1bb/controllers/performanceprofile_controller_test.go#L121

Reply:
Super cool! on it!

Reply:
we can (and probably should) postponing the event tests to a separate PR (or PRs, considering also the kubeletconfig controller needs work)

Reply:
OK i'll make sure to issue a separate PR for both controllers

Reply:
I can handle the kubeletconfig controller part, no worries

Reply:
here: https://github.com/openshift-kni/numaresources-operator/pull/86